Code/Tools/Libs/GuiFoundation/NodeEditor/Implementation/Connection.cpp

```diff
@@ -91,6 +91,25 @@ void ezQtConnection::UpdateGeometry()
     p.moveTo(startPoint);
     p.lineTo(endPoint);
   }
+  else if (pScene->GetConnectionStyle() == ezQtNodeScene::ConnectionStyle::SubwayLines)
+  {
+    // Used to enforce a small padding connection from the node itself to help readability.
+    const float fPaddingFromNode = 20.0f;
+
+    QPointF startPoint = m_OutPoint;
+
+    QPointF offsetStartPoint = QPointF(startPoint.x() + fPaddingFromNode, startPoint.y());
+
+    QPointF endPoint = m_InPoint;
+    QPointF offsetEndPoint = QPointF(endPoint.x() - fPaddingFromNode, endPoint.y());
+
+    p.moveTo(startPoint);
+    p.lineTo(offsetStartPoint);
+    DrawSubwayPath(p, offsetStartPoint, offsetEndPoint);
+    p.lineTo(offsetEndPoint);
+    p.moveTo(offsetEndPoint);
```

Comment:
This additional moveTo is unnecessary and causes artifacts since the lines are not properly connected at this point.

Reply:
Fixed